src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -102,10 +108,112 @@ def validate_role(self, role):
         return role
 
 
+class OrganizationMemberIndexQueryParamSerializer(serializers.Serializer):
+    email = serializers.CharField(
+        required=False,
+        default=None,
+        help_text="an email address to filter by.",
+    )
+    scope = serializers.ChoiceField(
+        required=False,
+        choices=settings.SENTRY_SCOPES,
+        help_text="filter on members with a specific scope.",
+    )
+
+    role = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        required=False,
+        help_text="filter on members with a role.",
+    )
+    isInvited = serializers.BooleanField(
+        required=False, help_text="filter on if the member is in an invited state."
+    )
+    ssoLinked = serializers.BooleanField(
+        required=False, help_text="filter on members who have linked an SSO identity."
+    )
+    has2fa = serializers.BooleanField(
+        required=False, help_text="filter on members who have 2fa enabled."
+    )
+    hasExternalUsers = serializers.BooleanField(
+        required=False, help_text="filter on members who have an external user linked."
+    )
+    query = serializers.CharField(
+        required=False,
+        help_text="filter by an email address, but look at secondary email's and primary account emails as well.",
+    )
+    expand = serializers.MultipleChoiceField(
+        choices=["externalUsers"],
```

Comment:
i'm wondering if it would be beneficial to define the expand fields explicitly as a property on our serializers, so that way they could be referenced via code here.